tsfresh/utilities/distribution.py

```diff
@@ -494,6 +494,84 @@ def close(self):
         self.pool.join()
 
 
+class RayDistributor(IterableDistributorBaseClass):
+    def __init__(
+        self,
+        address=None,
+        rayinit_config={},
+        n_workers=1,
+        disable_progressbar=False,
+        progressbar_title="Feature Extraction",
+    ):
+        """
+        Creates a new RayDistributor instance
+
+        :param address: the ip address and port number of the Ray Cluster
+        :type address: str
+        :param rayinit_config: external config for the ray.init calling
+        :type rayinit_config: dict
+        :param n_workers: How many workers should the multiprocessing pool have?
+        :type n_workers: int
+        :param disable_progressbar: whether to show a progressbar or not.
+        :type disable_progressbar: bool
+        :param progressbar_title: the title of the progressbar
+        :type progressbar_title: basestring
+        """
+        import ray
+
+        ray.init(address=address, **rayinit_config)
+        self.n_workers = n_workers
```

Comment:
If possible, I would recommend the following: as the number of worker property currently is not changing the cluster deployment, I would prefer if it is filled automatically. If this is not possible, we should remove the default value of 1 and maybe rename the parameter to make sure users know they need to set it to the number of cluster workers.
Now, it might look to users as if they can control the number of workers in ther cluster using this variable (which I think they can not)